src/client/js/Utils/Exporters.js

```diff
@@ -0,0 +1,82 @@
+/*globals define*/
+/*jshint node: true, browser: true, bitwise: false*/
+
+/**
+ * @author kecso / https://github.com/kecso
+ */
+define(['js/Utils/SaveToDisk', 'js/Loader/ProgressNotification'], function (saveToDisk, ProgressNotification) {
+    'use strict';
+
+    function exportProject(client, logger, projectParams, withAssets, callback) {
+        var progress = ProgressNotification.start('<strong>Exporting </strong> project ...');
+
+        client.exportProjectToFile(
+            projectParams ? projectParams.projectId : client.getActiveProjectId(),
+            projectParams ? projectParams.branchName : client.getActiveBranchName(),
+            projectParams ? projectParams.commitHash : client.getActiveCommitHash(),
+            withAssets,
+            function (err, result) {
+                clearInterval(progress.intervalId);
+                if (err) {
+                    logger.error('unable to save project', err);
+                    progress.note.update({
+                        message: '<strong>Failed to export: </strong>' + err.message,
+                        type: 'danger',
+                        progress: 100
+                    });
+                } else {
+                    progress.note.update({
+                        message: '<strong>Exported </strong> project <a href="' +
+                        result.downloadUrl + '" target="_blank">' + result.fileName + '</a>',
+                        progress: 100,
+                        type: 'success'
+                    });
+
+                    saveToDisk.saveUrlToDisk(result.downloadUrl);
+                }
+
+                if (typeof callback === 'function') {
+                    callback(err);
+                }
+            }
+        );
+    }
+
+    function exportModels(client, logger, selectedIds, callback) {
```

Comment:
It seemed too overcrowded to have multiple options (in the Model editor's context menu) for exporting models, but we might add the option on the ObjectBrowser and have the 'default' true in the model-editor